src/compile/unit.ts

```diff
@@ -79,6 +79,8 @@ export class UnitModel extends ModelWithField {
   public readonly selection: SelectionParameter[] = [];
   public children: Model[] = [];
 
+  public label: {mark: LabelMark; level: number}[] = [];
```

Comment:
This label property sounds weird.  

- It's singular, but it's an array.
- What's the `level` really for? I think we can name the `level` property to be more descriptive, or at least add some comments.

Reply:
Seeing down the `getLabel` never returns array with more than one item. Thus, I wonder if this `label` property really needs to be an array.